downloads/swan-lake-release-notes/swan-lake-alpha3/RELEASE_NOTE.md

```diff
@@ -591,7 +591,7 @@ websubhub:ContentDistributionMessage msg = {content: "This is sample content del
 var publishResponse = hubClientEP->notifyContentDistribution(msg);
 ```
 
-- Introduced an websubhub-listener configuration to configure a websubhub listener. 
+- Introduced an `websubhub-listener` configuration to configure a websubhub listener.
```

Comment:
Let's stick to the `websubhub-listener` instead of an because we have used 'the' elsewhere.